html/templates/header.html

```diff
@@ -37,7 +37,7 @@
 		<td valign='top'>
 			<font size='2'><b>Current Head Developers:</b> JamieD12<br></font>
 			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, monster860, Partheo, Wejengin2<br></font>
+			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, cuackles, Djiq, Ling, Manatee, Molti, monster860, OrcaCora, Partheo, Wejengin2, ynot01, Anerisa, Halcyon, TheReddDragon<br></font>
```

Comment:
```suggestion
			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, Cuackles, Djiq, Ling, Manatee, Molti, monster860, OrcaCora, Partheo, Wejengin2, ynot01, Anerisa, Halcyon, TheReddDragon<br></font>
```
Capitalization correction

Reply:
cuackles has his name set to lowercase on discord

Reply:
> cuackles has his name set to lowercase on discord

I have asked him how he prefers it and he says capital